CMSgov/onemac
# OY2-10715: make footer stick to bottom of viewport if content is too short to fill

Story: https://qmacbis.atlassian.net/browse/OY2-10715
Endpoint: TBD

### Details

Don't show a white bar below the footer when viewing pages with small content.

### Changes

- Set a minimum height on the main container
- Expand pre-footer content to fill the space

### Implementation Notes

- Needed to change the DOM hierarchy a little to accommodate this model without sacrificing responsiveness

### Test Plan

1. Sign in as `statesubmitteractive`. Verify that the Dashboard still flows below the fold.
2. Click the "Create New Submission" button. Verify that on the first page of the selection workflow, the footer is at the bottom of the viewport despite the preceding content being pretty short.
3. Sign in as `cmsreviewerunregistered`. Verify that role selection page exhibits the same behavior as seen in step 2.
4. Select "CMS Reviewer". Verify that the Group & Division page also does this.
